__tests__/environment-test.js

```diff
@@ -0,0 +1,125 @@
+"use strict"
+
+jest.dontMock('../environment.js');
+
+describe('environment', () => {
+  it('uses original config attributes if no env key is found', () => {
+    let environment = require('../environment.js').environment;
+
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      key:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      key:'fromDefaultConfig'
+    });
+  })
+
+  it('replaces the config attributes if an env key is found', () => {
+    let environment = require('../environment.js').environment;
+
+    process.env.A_KEY_NAME = 'fromEnv'
+
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      aKeyName:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      aKeyName:'fromEnv'
+    });
+  })
+
+  it('only inserts underscores for every new word', () => {
+    let environment = require('../environment.js').environment;
+
+    process.env.A_KEY_NAME_ID = 'fromEnv'
+
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      aKeyNameID:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      aKeyNameID:'fromEnv'
```

Comment:
Thanks! I've added [a test](https://github.com/facebook/mention-bot/pull/164/commits/993b8974037004c3f5803824c999d7b089f0f513#diff-3a7d1ff4aa9a51d872847b1ff3a1d80aR125). Let me know if that covers it.